### PR TITLE
Add content to file

### PR DESCRIPTION
**Pull request by obed300**
### In this project, I add validations to my models and have create specs for them.

### CODE REVIEW

**Comment by Kamba56** [comment]
Hi @obed300 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that **has linters errors. Kindly fix all linters errors**

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: spec/post_spec.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  it 'title should not exceed 250 characters' do
+    subject.comments_counter = 'a'
+    expect(subject).to_not be_valid
+  end
+
+  it 'title should not exceed 250 characters' do
+    subject.likes_counter = 'a'
+    expect(subject).to_not be_valid
+  end
```
Comment (VICTOR-CHUKWU) [inline spec/post_spec.rb:26]: - [ ] kindly add test for your methods like `five_recent_comments` and  `update_post_counter`

File: spec/user_spec.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+
+  it 'must be an integer greater than or equal to zero' do
+    subject.posts_counter = 'a'
+    expect(subject).to_not be_valid
+  end
+end
```
Comment (VICTOR-CHUKWU) [inline spec/user_spec.rb]: - [ ] kindly add test for `three_most_recent` methods

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @obed300  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

- [ ] kindly add spec for likes and comment

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
Hi @obed300, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯
3. All tests are passing 👍 
4. Well-written code 👍 

## Optional suggestions
No optional suggestions :)

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**